_config.yml

```diff
@@ -1 +1 @@
-exclude: [src]
+exclude: ["src","highlight","hooks"]
```

Comment:
"highlight" should not be excluded here; it contains the javascript that we still use for this version